src/hypofuzz/hypofuzz.py

```diff
@@ -384,49 +391,56 @@ def _replace_metadata(self, metadata: Metadata) -> None:
         db = get_db()
         db.replace_metadata(self.database_key, metadata)
 
+    @property
+    def _phase(self) -> Phase:
+        phase = Phase.GENERATE
+        if self._replay_queue:
+            phase = Phase.REPLAY
+        elif self.pool._in_distill_phase:
+            phase = Phase.DISTILL
+        elif self.pool.interesting_examples:
+            phase = Phase.SHRINK if self.shrinking else Phase.FAILED
+        return phase
+
```

Comment:
I went for `phase` over `how_generated` because a report is an aggregate unit, so I want to make claims about the current phase, not that all examples in the report were generated a particular way

Reply:
oh, but they *are* all generated the same way, because we emit a report whenever we switch phases. This is what lets us linearize. I will probably go back to `how_generated`